tools/wptrunner/wptrunner/browsers/safari.py

```diff
@@ -60,6 +61,66 @@ def env_options():
     return {}
 
 
+def run_info_extras(**kwargs):
+    webdriver_binary = kwargs["webdriver_binary"]
+    safari_bundle, safari_info = get_safari_info(webdriver_binary)
```

Comment:
So this can fail if you don't find the Plist in the expected paths. It would be nice to handle that explictly rather than just raising an exception trying to unpack None as a tuple.